### ISSUE TITLE
Should "Infinite Harmonics" checkbox be available when on "Sinusoidal"?

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/fourier-making-waves#180]
Reported by @kathy-phet in https://github.com/phetsims/fourier-making-waves/issues/178#issuecomment-926970316: 
> Design question (@arouinfar) - Should "Infinite Harmonics" checkbox be available when on "Sinusoidal". It feels like it should be disabled for Sinusoidal, just like it is for custom.

**Comment by pixelzoom** [comment]
My recollection from the Java sim design (supported by comments in the Java code) is that showning "Infinite Harmonics" was considered valuable for "sinusoidal".  It shows the student that the "sinusoid" preset is identical to a single harmonic.

**Comment by kathy-phet** [comment]
I guess you could rationalize this as "even if you had infinite harmonics to make up your wave, sinusoidal is one harmonic set to a non-zero value and all the rest set to 0". I can see that now, it was just messing with my brain to think about one harmonic being infinite harmonics. So I can now rationalize either way here. 

@arouinfar - what do you think? did you see anything in interviews around this that might provide insight into what students thought about this?

**Comment by pixelzoom** [comment]
I agree that there's a dissonance here between "infinite" and "only one".   

Rather than removing this feature for "sinusoidal", you might consider changing the "Infinite Harmonics" label to something that indicates "show me the exact waveform that we're trying to approximate". 

In deciding whether to remove this feature for "sinusoidal", one thing to consider is whether teachers (or PhET?) might have scaffolding that uses this feature.

**Comment by arouinfar** [comment]
My vote would be to keep the "Infinite Harmonics" label and disable it for the sinusoidal option. 

I think the goal of this checkbox is to show the waveform created from an infinite number of harmonics. As you add more harmonics, the sum looks more like the infinite harmonics waveform. If the goal is to demonstrate that the "sinusoidal" preset is identical to a single harmonic, it needs teacher facilitation. Two of the interviewees found the "Infinite Harmonics" checkbox early on while still exploring the sinusoidal waveform. Neither one knew what to make of it until exploring one of the other options, such as the square.

> In deciding whether to remove this feature for "sinusoidal", also consider that teachers (and PhET?) might have scaffolding that uses this feature.

I looked through several activities, including those created by Trish and Sam, and did not find any mention of the Infinite Harmonics option aside from this one:
>Try out the different options in the “Select Function” pull-down menu.  For each function, try increasing and decreasing the number of harmonics.  What effect does this have and why?  What would happen if you had many more than 11 harmonics?  Test your prediction by checking the box labeled, “Show function with infinite number of harmonics.”

@pixelzoom let's proceed with disabling the Infinite Harmonics checkbox when sinusoidal is selected.

**Comment by pixelzoom** [comment]
'Infinite Harmonics' checkbox is now disabled for 'sinusoid'. @arouinfar please review in master.

**Comment by arouinfar** [comment]
Thanks @pixelzoom. Looks good in master.
